chapters/classes.tex

```diff
@@ -601,6 +601,8 @@ \subsection{Short Class Definitions}\label{short-class-definitions}
 end IDENT1;
 \end{lstlisting}
 
+An exception to the above is that if the short class definition is declared as \lstinline!encapsulated!, then the modifiers cannot be looked up in the enclosing scope.
```

Comment:
Does it mean that they should be looked up in the global scope? Should they follow the same rule as for the encapsulated classes (only predefined types, functions and operators defined in the specification or names starting with .)?

Reply:
I'll try to clarify it.